homeassistant/components/systemmonitor/sensor.py

```diff
@@ -790,17 +732,18 @@ def clean_obsolete_entities() -> None:
     async_add_entities(entities)
 
 
-class SystemMonitorSensor(CoordinatorEntity[MonitorCoordinator[dataT]], SensorEntity):
+class SystemMonitorSensor(CoordinatorEntity[SystemMonitorCoordinator], SensorEntity):
     """Implementation of a system monitor sensor."""
 
     _attr_has_entity_name = True
     _attr_entity_category = EntityCategory.DIAGNOSTIC
-    entity_description: SysMonitorSensorEntityDescription[dataT]
+    entity_description: SysMonitorSensorEntityDescription
+    argument: str
 
     def __init__(
```

Comment:
Maybe add an `async_added_to_hass` to tell the `self.async_on_remove(coordinator.async_subscribe_update_types(XXX))` to flip on updates for that area so it only does the initial refresh of everything, and than switches to updating what is requested